dm/syncer/safe_mode.go

```diff
@@ -30,6 +36,17 @@ func (s *Syncer) enableSafeModeByTaskCliArgs(tctx *tcontext.Context) {
 }
 
 func (s *Syncer) enableSafeModeInitializationPhase(tctx *tcontext.Context) {
+	var err error
+	defer func() {
+		if err != nil {
+			// send error to the fatal chan to interrupt the process
+			s.runFatalChan <- unit.NewProcessError(err)
+		}
+		if !s.safeMode.Enable() {
+			s.tctx.L().Warn("enable safe-mode failed")
```

Comment:
maybe we should not write log as "failed", because when checkpoint equals safeModeExitPoint it's expected

Reply:
If so, maybe remove it is better